tests/mocks.py

```diff
@@ -31,10 +32,21 @@ def forward(self, indices: Optional[torch.LongTensor] = None) -> torch.FloatTens
 class MockEvaluator(Evaluator):
     """A mock evaluator for testing early stopping."""
 
-    def __init__(self, losses: Iterable[float], automatic_memory_optimization: bool = True) -> None:
```

Comment:
@mberr seems like this is causing issues. Can we split this into a separate PR and revert?

Reply:
29be8f0